src/pages/TopUp/components/WalletConnectModal.tsx

```diff
@@ -88,11 +88,16 @@ const WalletConnectModal: React.FC<{
             <FormattedMessage
               defaultMessage="Connect to {network}"
               values={{
-                network: IS_MAINNET
-                  ? 'Ethereum mainnet'
-                  : `${EL_TESTNET_NAME.toLowerCase()} testnet`,
+                network: IS_MAINNET ? (
```

Comment:
Seems like we duplicate this exact same logic in a few places - I wonder if we could make this more DRY?

Reply:
Agree with this sentiment... Just added some commits implementing a `useIntlNetworkName` hook to remove a lot of the redundancy here.

Can access using `const { executionLayerName, consensusLayerName } = useIntlNetworkName();`